spec.bs

```diff
@@ -260,8 +260,21 @@ dictionary AuctionAdInterestGroupKey {
 The <dfn for=Navigator method>leaveAdInterestGroup(group)</dfn> method steps
 are:
 
+1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the
+  "[=join-ad-interest-group=]" [=policy-controlled feature=], then [=exception/throw=] a
+  "{{NotAllowedError}}" {{DOMException}}.
+1. Let |ownerUrl| be the result of running the [=URL parser=] on
+    |group|["{{AuctionAdInterestGroupKey/owner}}"].
+1. If |ownerUrl| is an error [=exception/throw=] a {{TypeError}}.
+1. Set |owner| to |ownerUrl|'s [=url/origin=].

```

Comment:
Where is |owner| defined? Same for |name| just below.

Reply:
Changed above lines from "Set" to "Let".